### ISSUE TITLE
Editor compatible issue in MacOS Catalina 10.15.1

### ISSUE THREAD

**Issue by yongyili** [rlabrecque/Steamworks.NET#312]
I have upgraded my iMac Pro to Catalina, then I cannot run my game in Unity3d editor.
The error popup window is: 
![image](https://user-images.githubusercontent.com/2041418/68006523-c66eb380-fcb3-11e9-9d2a-22d074b9590e.png)

MacOS 10.15 have dropped 32bit support, I do not know whether it is the root cause.

**Comment by bartofzo** [comment]
Having the same issue, unable to work on a project using Steamworks.NET on macOS Catalina. Did anyone manage to solve this?

**Comment by rlabrecque** [comment]
Can you try copying libsteam_api.dylib out of the steam_api.bundle? Place it next to the bundle, and then remove the bundle. Then restart the Unity editor.

If you're on a version of Unity older than 2019.3 (or so) you may need to rename libsteam_api.dylib to steam_api.bundle.

Does that help at all?

**Comment by joonjoonjoon** [comment]
I'm facing the same issue, and tried what you said on 2019.3.1f1.
- I re-imported everything, and kept your folder structure (meaning the bundle file is just inside the Plugins folder).
- I grabbed the "libsteam_api.dylib" file and put it inside the Plugins folder, and deleted the bundle.
- It no longer prompts the popup like in the image of the original post, but it still doesn't work. 
- I get this error:

`[Steamworks.NET] Could not load [lib]steam_api.dll/so/dylib. It's likely not in the correct location. Refer to the README for more details.
System.DllNotFoundException: steam_api
  at (wrapper managed-to-native) Steamworks.NativeMethods.SteamAPI_RestartAppIfNecessary(Steamworks.AppId_t)
  at Steamworks.SteamAPI.RestartAppIfNecessary (Steamworks.AppId_t unOwnAppID) [0x00007] in /Users/joon/git/nuts/Assets/Plugins/Steamworks.NET/Steam.cs:72 
  at SteamManager.Awake () [0x0007f] in /Users/joon/git/nuts/Assets/Plugins/Steamworks.NET/SteamManager.cs:85 `

**Comment by rlabrecque** [comment]
Hey @joonjoonjoon , thanks so much for the info!

Could you actually try this again by downloading the latest commit, and see if the problem still occurs without manually doing chmod?

Hopefully fixed with https://github.com/rlabrecque/Steamworks.NET/commit/b934ca0452f628c44ed0ec4ef667fa280ce57bf2

**Comment by MegaDwarfDan** [comment]
We are still encountering this issue on the latest Steamworks release.
We are running Unity 2018.4.22f1 and Catalina on the Mac.

We've also tried this:
"Can you try copying libsteam_api.dylib out of the steam_api.bundle? Place it next to the bundle, and then remove the bundle. Then restart the Unity editor.

If you're on a version of Unity older than 2019.3 (or so) you may need to rename libsteam_api.dylib to steam_api.bundle."

as well as this:
"Running this command on the .bundle file fixed it for me (make sure the path is correct for your situation)
chmod -R a+xr "Assets/Plugins/Steamworks.NET/steam_api.bundle""

And neither of them solved our problem. We're looking for any possible solution to this problem.
Thanks!

**Comment by joonjoonjoon** [comment]
there's another thing I think we needed to do:
next time you see the popup, open up "Security & Privacy" in preferences.

on either the general or privacy tab, you will see something mentioned about permissions that you need to accept.
(you might have to select "Move to trash" once for it to show up)

**Comment by MegaDwarfDan** [comment]
We managed to get past the issue yesterday, but I'm not sure what actually fixed it. We followed all the same steps, but deleted the Library folder before doing anything else, and that ended up working. But it really could've been any of the individual steps just deciding to work this time. Unity is finicky like that sometimes. I'll post again if I get better steps to reproduce success on my end.

And I'll check out the Security & Privacy in preferences, thanks Joon!
